e2e/atlas/users_test.go

```diff
@@ -53,4 +53,25 @@ func TestUsers(t *testing.T) {
 			t.Fatalf("expected len(users) > 0, got %v", len(users))
 		}
 	})
+
+	t.Run("Describe", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			iamEntity,
+			usersEntity,
+			"describe",
+			"nonExistentUser",
```

Comment:
this is going to error, first on usage, and then the server is going to return 404